spec/index.bs

```diff
@@ -1113,6 +1113,41 @@ Given an {{HTMLScriptElement}} (|script|), this algorithm performs the following
 
     If the algorithm threw an error, rethrow the error.
 
+## Get Trusted Types-compliant attribute value ## {#validate-attribute-mutation}
+To <dfn abstract-op export>get Trusted Types-compliant attribute value</dfn> on {{Attr}} |attribute| with {{Element}} |element| and {{TrustedType}} or a string |newValue|, perform the following steps:
+
+1. Let |expectedType| be null.
+1. Let |sink| be null.
+1. Find the row in the following table, where |element| is in the first column, and |attribute|'s <a for="Attr">namespace</a> and <a for="Attr">local name</a> match the values in the second and third column, respectively.
+    If a matching row is found, set |expectedType| and |sink| to the value of the fourth, and fifth column in that row, respectively.
+
+    <table>
+      <thead>
+        <tr><th>Element<th>Attribute namespace<th>Attribute local name<th>TrustedType<th>Sink
+      <tbody>
+        <tr><td>{{HTMLIFrameElement}}<td>null<td>"srcdoc"<td>{{TrustedHTML}}<td>"HTMLIFrameElement srcdoc"
+        <tr><td>{{HTMLEmbedElement}}<td>null<td>"src"<td>{{TrustedScriptURL}}<td>"HTMLEmbedElement src"
+        <tr><td>{{HTMLScriptElement}}<td>null<td>"src"<td>{{TrustedScriptURL}}<td>"HTMLScriptElement src"
+        <tr><td>{{HTMLObjectElement}}<td>null<td>"data"<td>{{TrustedScriptURL}}<td>"HTMLObjectElement data"
+        <tr><td>{{HTMLObjectElement}}<td>null<td>"codebase"<td>{{TrustedScriptURL}}<td>"HTMLObjectElement codebase"
+        <tr><td>{{SVGScriptElement}}<td>null<td>"href"<td>{{TrustedScriptURL}}<td>"SVGScriptElement href"
+        <tr><td>{{SVGScriptElement}}<td><a>XLink namespace</a><td>"href"<td>{{TrustedScriptURL}}<td>"SVGScriptElement href"
```

Comment:
Given https://searchfox.org/mozilla-central/rev/76cb3efe3b19e649bf675bb6ec5d4af8109b9771/testing/web-platform/tests/trusted-types/block-string-assignment-to-attribute-via-attribute-node.html#15,36 and https://jsfiddle.net/3vef62r9/1/ `onclick` should be handled here too.

CC @koto

Reply:
These should be handled in https://w3c.github.io/trusted-types/dist/spec/#enforcement-in-event-handler-content-attributes instead (via HTML integration).

Reply:
Perhaps other (all?) event handlers too.

Reply:
Event handlers are intentionally not in this list, they are controlled via HTML integration. https://w3c.github.io/trusted-types/dist/spec/#enforcement-in-event-handler-content-attributes